services/QuillLMS/client/app/bundles/PremiumHub/containers/UsageSnapshotsContainer.tsx

```diff
@@ -145,6 +231,15 @@ const UsageSnapshotsContainer = ({ adminInfo, accessType, }) => {
 
   function handleSetSelectedTabFromDropdown(option) { setSelectedTab(option.value) }
 
+  function selectedGradesToPass() {
+    // we need to pass the backend an empty array when all grades are selected so we include data from classrooms that do not have a grade
+    if (!selectedGrades) { return [] }
+
+    // we need to pass the backend an empty array when all grades are selected so we include data from classrooms that do not have a grade
```

Comment:
Duplicate comment.

Also, side note: we no longer need to do this.  You can pass the string `"null"` to filter for those grades now.  (The value for the new "No grade set" option.)  That said, we can also keep doing this if it keeps things simpler.